docs/docs/ko/guides/develop/projects/manifests.md

```diff
@@ -54,11 +53,11 @@ let workspace = Workspace(
 
 A question that often comes up is whether to use a single project or multiple projects in a workspace. In a world without Tuist where a mono-project setup would lead to frequent Git conflicts the usage of workspaces is encouraged. However, since we don't recommend including the Tuist-generated Xcode projects in the Git repository, Git conflicts are not an issue. Therefore, the decision of using a single project or multiple projects in a workspace is up to you.
 
-In the Tuist project we lean on mono-projects because the cold generation time is faster (fewer manifest files to compile) and we leverage [project description helpers](/ko/guides/develop/projects/code-sharing) as a unit of encapsulation. However, you might want to use Xcode projects as a unit of encapsulation to represent different domains of your application, which aligns more closely with the Xcode's recommended project structure.
+In the Tuist project we lean on mono-projects because the cold generation time is faster (fewer manifest files to compile) and we leverage [project description helpers](/en/guides/develop/projects/code-sharing) as a unit of encapsulation. However, you might want to use Xcode projects as a unit of encapsulation to represent different domains of your application, which aligns more closely with the Xcode's recommended project structure.
 
 ## Config.swift
 
-Tuist provides [sensible defaults](/ko/contributors/principles.html#default-to-conventions) to simplify project configuration. However, you can customize the configuration by defining a [`Config.swift`](/ko/references/project-description/structs/config) manifest under the `Tuist` directory, which is used by Tuist to determine the root of the project.
+Tuist provides [sensible defaults](/en/contributors/principles.html#default-to-conventions) to simplify project configuration. However, you can customize the configuration by defining a [`Config.swift`](/en/references/project-description/structs/config) manifest under the `Tuist` directory, which is used by Tuist to determine the root of the project.
```

Comment:
Why are we changing the references to English in the Korean section of the docs?

Reply:
I need to make the links locale independent. Crowding doesn't know about it and copies the links as they are.